### PR TITLE
add modes trunks and list-trunks

### PR DESCRIPTION
**Pull request by cgagnaire**
Add modes trunks and list-trunks to F5 BigIP plugin.

### CODE REVIEW

**Comment by Sims24** [comment]
@cgagnaire my review is about why do we have to specify --regexp option. Why do not use regexp by default as we do recently for other devices ? Any technical reason on f5 size or oid values ?

File: network/f5/bigip/snmp/mode/listtrunks.pm [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+        next if ($oid !~ /^$sysTrunkName\.(.*)$/);
+        my $instance = $1;
+
+        # Get all without a name
+        if (!defined($self->{option_results}->{name})) {
+            push @{$self->{trunks_selected}}, $instance; 
+            next;
+        }
+        
+        if (!defined($self->{option_results}->{use_regexp}) && $self->{result}->{$sysTrunkName . '.' . $instance} eq $self->{option_results}->{name}) {
```
[the remark this answers is not in the document]
Comment (cgagnaire) [reply network/f5/bigip/snmp/mode/listtrunks.pm]: Yes, I could have use filter_name option, but was wrongly inspired by other list modes from this plugin.

[the remark this answers is not in the document]
Comment (Sims24) [reply network/f5/bigip/snmp/mode/listtrunks.pm]: Ok, as they are new modes and we do not risk to break anything please make it "standard"

**Comment by Sims24** [comment]
Ok let's merge this to avoid unifying all the f5 plugin stack.

**Pull request merged**